acquia-http-hmac.gemspec

```diff
@@ -13,9 +13,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.executables << 'acq-http-request'
 
-  s.add_runtime_dependency('excon', '~> 0.44', '>= 0.44.4')
```

Comment:
I'd add it as a dev dependency so you can test it, but end users must have it installed if they want to use it.

Sequel uses the same pattern (if you don't install `mysql2` and try to use Sequel with `mysql2://` you get a require error).